File: app/src/main/java/kr/ac/korea/translator/model/TextContainer.java
package kr.ac.korea.translator.model;

import lombok.Data;

@Data
public class TextContainer {
    int x;
    int y;
    int w;
    int h;
    String rst;

    public TextContainer(Integer[] pos, String text){
        this.x = pos[0];
        this.y = pos[1];
        this.w = pos[2];
        this.h = pos[3];
        this.rst=text;
    }
}

File: app/src/main/java/kr/ac/korea/translator/model/TextContainer.kt
package kr.ac.korea.translator.model

data class TextContainer(var pos: Array<Int?>, var text: String?) {
    var x: Int = pos[0]!!
    var y: Int = pos[1]!!
    var w: Int = pos[2]!!
    var h: Int = pos[3]!!
    var rst: String? = text
}